otherlibs/configurator/src/v1.ml

```diff
@@ -532,28 +528,51 @@ const char *s%i = "BEGIN-%i-false-END";
   let extract_values obj_file vars =
     let values =
       Io.with_lexbuf_from_file obj_file ~f:(Extract_obj.extract [])
-      |> Int.Map.of_list_exn
+      |> List.fold_left ~init:Int.Map.empty ~f:(fun acc (key, v) ->
+             Int.Map.update acc ~key ~f:(function
+               | None -> Some (Ok v)
+               | Some (Error vs) -> Some (Error (v :: vs))
+               | Some (Ok v') ->
+                 Some
+                   ( if v = v' then
+                     Ok v'
+                   else
+                     Error [ v; v' ] )))
```

Comment:
Can simplify by using `string list Int.Map.t` instead of `(string, string list) Result.t Int.Map.t`.

Reply:
done